docs/pages/guides/design-system-cheat-sheet.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Design System Utilities
+keywords: design system cheat sheet
+sidebar: left-navigation-sidebar
+toc: false
+permalink: design-system-utilities.html
+folder: guides
+summary: The following guide covers all the major utility features of the Fundamental UI (FDUI) design system toolkit. We recommend that you utilize these functions, mixins and helper classes to maintain consistency throughout your application. Using the toolkit features will ensure that you application will remain compatible and easy to upgrade with the future versions of FDUI, transition between themes and customization.
+---
```

Comment:
Include a blurb here about preferably using the mixins and functions with a link. http://localhost:4000/how-to-use-scss.html#core-functions-and-mixins